it/v1/endpoints/UnignoreBenefitControllerISpec.scala

```diff
@@ -30,58 +30,30 @@ import v1.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
 
 class UnignoreBenefitControllerISpec extends IntegrationBaseSpec {
 
-  private trait Test {
-
-    val nino: String          = "AA123456A"
-    val taxYear: String       = "2019-20"
-    val benefitId: String     = "b1e8057e-fbbc-47a8-a8b4-78d9f015c253"
-    val correlationId: String = "a1e8057e-fbbc-47a8-a8b4-78d9f015c253"
-
-    val hateoasResponse: JsValue = Json.parse(
-      s"""
-         |{
-         |   "links":[
-         |      {
-         |         "href":"/individuals/state-benefits/$nino/$taxYear?benefitId=$benefitId",
-         |         "rel":"self",
-         |         "method":"GET"
-         |      },
-         |      {
-         |         "href":"/individuals/state-benefits/$nino/$taxYear/$benefitId/ignore",
-         |         "rel":"ignore-state-benefit",
-         |         "method":"POST"
-         |      }
-         |   ]
-         |}
-       """.stripMargin
-    )
-
-    def uri: String = s"/$nino/$taxYear/$benefitId/unignore"
-
-    def desUri: String = s"/income-tax/state-benefits/$nino/$taxYear/ignore/$benefitId"
-
-    def setupStubs(): StubMapping
+  "Calling the 'unignore benefit' endpoint" should {
+    "return a 200 status code" when {
+      "any valid request is made" in new NonTysTest {
 
-    def request(): WSRequest = {
-      setupStubs()
-      buildRequest(uri)
-        .withHttpHeaders(
-          (ACCEPT, "application/vnd.hmrc.1.0+json"),
-          (AUTHORIZATION, "Bearer 123") // some bearer token
-        )
-    }
+        override def setupStubs(): StubMapping = {
+          AuditStub.audit()
```

Comment:
Can move the following to the request method to prevent repeating code.

"         AuditStub.audit()
          AuthStub.authorised()
          MtdIdLookupStub.ninoFound(nino)
"

You will need to change the return type of setupStubs to Unit